---
applied_at: '2022-11-30'
applies_to:
  - sdks
  - python
is_impactful: false
is_new_feature: false
release_source_url: 'https://github.com/box/box-python-sdk/releases/tag/v3.5.1'
collapse: true
---

# Box Python SDK `v3.5.1` released

### Bug Fixes

* Renew connection when Connection reset error occurs ([#771][1]) ([`bcaab27`][2]), closes [#756][3] [#757][4] [#763][5] [#765][6] [#766][7] [#770][8]
* Retry JWT auth when got error: required unique `jti` claim. ([#768][9]) ([`878e958`][10])
* Update `pyjtw` dependency to work with Python 3.10 ([#772][11]) ([`b13c5cd`][12])

[1]: https://github.com/box/box-python-sdk/issues/771

[2]: https://github.com/box/box-python-sdk/commit/bcaab277c3cabba498076d066366abbaa5507904

[3]: https://github.com/box/box-python-sdk/issues/756

[4]: https://github.com/box/box-python-sdk/issues/757

[5]: https://github.com/box/box-python-sdk/issues/763

[6]: https://github.com/box/box-python-sdk/issues/765

[7]: https://github.com/box/box-python-sdk/issues/766

[8]: https://github.com/box/box-python-sdk/issues/770

[9]: https://github.com/box/box-python-sdk/issues/768

[10]: https://github.com/box/box-python-sdk/commit/878e958abfb01740656aaff42492777753e4c8ea

[11]: https://github.com/box/box-python-sdk/issues/772

[12]: https://github.com/box/box-python-sdk/commit/b13c5cd34105d3f774d3f6d35db7aaf51dd3e247
